src/filtering/update.jl

```diff
@@ -121,3 +133,38 @@ function update!(
 
     return x_out
 end
+
+
+# Kronecker version
+function update!(
+    x_out::SRGaussian{T,<:Kronecker.KroneckerProduct},
```

Comment:
As for the others: check that this is the correct type, and also check that the left factor is all identity